week4-ts/src/components/StoreCard.tsx

```diff
@@ -0,0 +1,74 @@
+import styled from "styled-components";
+import { Store } from "../types";
+
+interface StoreCardProps {
+  store: Store;
+  isCheck: boolean;
+}
+
+export default function StoreCard({ store, isCheck }: StoreCardProps) {
+  const { place_url, place_name, phone, distance, address_name } = store;
+
+  return (
+    <StCard>
+      <a href={place_url || undefined}>{place_name}</a>
```

Comment:
`undefined` 옵셔널을 붙여준 이유가 궁금합니다 !!

```suggestion
      <a href={place_url}>{place_name}</a>
```

API 에 `place_url` 이 없는 것 떄문이라면 아래처럼 작성하는 게 더 명시적이지 않을까 합니다! (css 코드가 조금 변경될 수 있겠네요,!)

```suggestion
      {place_url && <a href={place_url}>{place_name}</a>}
```

Reply:
확인해보니까 불필요한 부분이네요... 꼼꼼하지 못했네요
만약 해당 데이터를 줄 수도 있고 안 줄수도 있는 경우라면 좜님이 밑에 써주신 코드로 작성하도록 할게요!👍